libweb3jsonrpc/AccountHolder.cpp

```diff
@@ -103,7 +103,8 @@ void AccountHolder::clearQueue(int _id)
 
 AddressHash SimpleAccountHolder::realAccounts() const
 {
-	return m_keyManager.accounts();
+	auto accounts = m_keyManager.accounts();
```

Comment:
This snippet appears twice at least - better to use a utility function? also note you can do `AddressHash() + accounts`